tests/models/abiotic/test_abiotic_model.py

```diff
@@ -368,12 +373,8 @@ def test_update_abiotic_model(dummy_climate_data, fixture_core_components):
     lyr_strct = fixture_core_components.layer_structure
 
     # initialise model
-    with (
-        patch_run_update("abiotic") as mock_update,
-        patch_bypass_setup("abiotic") as mock_bypass_setup,
-    ):
-        mock_update.return_value = False
-        mock_bypass_setup.return_value = False
+    with patch_static_config("abiotic") as mock_static_config:
+        mock_static_config.return_value = False, False
```

Comment:
And with the change to the patchers, this becomes:

```suggestion
    with patch_static_config(AbioticModel) as mock_static_config:
        mock_static_config.return_value = False, False
```

Reply:
Sorted in [e7689cd](https://github.com/ImperialCollegeLondon/virtual_ecosystem/pull/693/commits/e7689cdd8ad2b30f231a6d7a4182193a98e4bd64)